### ISSUE TITLE
Allow purpose icon attribute in manifest.js

### ISSUE THREAD

**Issue by AmauryD** [zonkyio/ember-web-app#627]
Hello,

I get a warning message when i use the purpose attribute in icons.

My manifest.js file (simplified):
```js
module.exports = function(/* environment, appConfig */) {
  return {
    ...
    icons: [
      ...
      {
        src: '/assets/favicons/maskable_icon.png',
        sizes: '512x512',
        type: "image/png",
        purpose: "maskable"
      }
  }
}
```

```WARNING: manifest.webmanifest validation: Unknown icon attribute "purpose".```

The property seems to be moved (at least Lighthouse is ok with it) to manifest.webmanifest but the warning message is a little confusing, purpose should be a valid attribute.

**Comment by bobisjan** [comment]
Hi :wave:,

this warning comes from `web-app-manifest-validator`, it looks that it is already fixed in `master` by san650/web-app-manifest-validator#8 😍.

I'll update and release patch version when available, until then this warning can be ignored, sorry for the confusion.

**Comment by san650** [comment]
I've released a new version of web-app-manifest-validator

**Comment by bobisjan** [comment]
Released as v5.0.1 🚀.
